### DISCUSSION TITLE
Commits not being showing in Contribution Graph

### DISCUSSION THREAD

**Discussion by talhameer** [community/community#129614] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

In my GitHub account, I have added multiple emails to the account. When I commit from an email specific to the organization, the commits are not listed in the contribution graph. Previously when I was using Linux Ubuntu OS this thing was working fine. When I switched the OS to Windows, my git global email on the system and primary email on GitHub were the same. Since I changed the git global email on the system, my contributions are not visible. 
For some repos, I've added the git local email same as I have the primary email in GitHub, and commits from those repos are visible.

### May help
One thing that I would like to mention here is that on Windows OS, I didn't create an SSH key for GitHub explicitly, but authenticated using VS Code and started cloning, committing, and pushing code to GitHub.

### Things I have done
* Create an SSH key explicitly and added to GitHub
* Verified that "Private Contributions" is enabled
* Verified Email spelling

**Comment by davevad93** [comment]
Hi @talhameer ,

There must have been some issue when you changed your global git email, take a look at this from the docs maybe you'll find something useful: 

- [Why are my contributions not showing up on my profile?](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile)

**Reply by talhameer** [reply]
Hey @davevad93, Thanks for sharing this document. One thing that could be an issue is that I'm committing in a feature branch. But this is also confusing because commits in feature branches in other repos from other Organizations are visible in the contribution graph.

**Reply by talhameer** [reply]
Yeah, I've confirmed that the commit in the master branch is added to the contribution graph. Now my questions are: 

1. When the feature branch is merged those commits will be added to the graph?
2. Why this behavior is not the same in every organization, e.g. owned by me and added as a member?

**Reply by davevad93** [reply]
Hey @talhameer , you're welcome:

1. Yes, when you merge a secondary `branch` into the `main` one, all the commits plus the merged `pull request` will be listed in the graph , e.g. if 7 `commits` are merged, the graph will list 8 total contributions.
2. I don't know exactly why, if you have correctly configured the `global user.email` and you own those organizations the contributions should appear as usual in the graph.
